stable/gitlab-ce/templates/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+{{- if .Values.ingress.annotations }}
+  annotations:
+{{ toYaml .Values.ingress.annotations | indent 4 }}
+{{- end }}
+  name: {{ template "fullname" . }}
```

Comment:
Sorry for the late feedback. With respect to the existing chart, it does. For new charts, the recommendation is to use `{{ template "name" . }}` for the `app` label.